azure-pipelines.yml

```diff
@@ -170,6 +170,64 @@ stages:
               summaryFileLocation: '$(System.DefaultWorkingDirectory)/**/coverage.xml'
               reportDirectory: '$(System.DefaultWorkingDirectory)/**/htmlcov'
 
+      - job: no_sqlalchemy
+        condition: or(eq(variables.isScheduled, true), eq(variables.isMain, true), eq(variables.isManual, true))
+
+        services:
+          postgres: postgres
+
+        variables:
+          GE_pytest_opts: ''
+
+        strategy:
+          matrix:
+            Python38:
+              python.version: '3.8'
+              GE_pytest_pip_opts: '--requirement requirements-dev.txt --constraint constraints-dev.txt'
+              pip_uninstall_packages: 'sqlalchemy'
+
+        steps:
+          - task: UsePythonVersion@0
+            inputs:
+              versionSpec: '$(python.version)'
+            displayName: 'Use Python $(python.version)'
+
+          - bash: python -m pip install --upgrade pip==21.3.1
+            displayName: 'Update pip'
+
+          - script: |
+              sudo apt-get install -y pandoc
+              pip install pypandoc
+            displayName: 'Install pandoc'
```

Comment:
More of a general question but what is pandoc doing here? I've seen it a few times and haven't figured out where it's used.

Reply:
I just copied from comprehensive - I'm not sure, and also the napoleon-docstrings and other params in the pytest command. I think these may be related to code coverage / old rst docs. At some point worth investigating and removing.